From 8d2fb473b5fb339b2d89c34106b46cec697863c0 Mon Sep 17 00:00:00 2001
From: Qiang Yu <Qiang.Yu@amd.com>
Date: Wed, 3 May 2017 14:13:42 +0800
Subject: [PATCH] amdgpu: [HYBRID] add AMDGPU_CAPABILITY_SSG_FLAG

This is for UMD be able to query SSG present.

Change-Id: Id52286984c8a43b77ad443799b267a6d0b23df54
Signed-off-by: Qiang Yu <Qiang.Yu@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
Reviewed-by: Chunming Zhou <david1.zhou@amd.com>
---
 include/drm/amdgpu_drm.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/drm/amdgpu_drm.h b/include/drm/amdgpu_drm.h
index cccb91d..f758548 100644
--- a/include/drm/amdgpu_drm.h
+++ b/include/drm/amdgpu_drm.h
@@ -951,6 +951,8 @@ struct drm_amdgpu_virtual_range {
 #define AMDGPU_CAPABILITY_PIN_MEM_FLAG		(1 << 0)
 /* query direct gma capability */
 #define AMDGPU_CAPABILITY_DIRECT_GMA_FLAG	(1 << 1)
+/* query ssg capability */
+#define AMDGPU_CAPABILITY_SSG_FLAG		(1 << 2)
 
 struct drm_amdgpu_capability {
 	uint32_t flag;
-- 
2.7.4

